### PR TITLE
Update creature.lua

### PR DESCRIPTION
**Pull request by EleanorRigby1**
It is not my proposal to remove it, the system is good if it works, I don't know much about programming, so I didn't fix it. But that corrected the error.

With that code, when you have White Skull and let the monsters attack you, you give it to exit and re-enter as in 20 seconds you will be kicked with white skull.

### CODE REVIEW

File: data/events/scripts/creature.lua [hunk 1]
```diff
@@ -75,29 +75,6 @@ function Creature:onTargetCombat(target)
 		end
 	end
 
-	if target:isPlayer() then
-		if self:isMonster() then
-			local protectionStorage = target:getStorageValue(Storage.combatProtectionStorage)
-
-			if target:getIp() == 0 then -- If player is disconnected, monster shall ignore to attack the player
-			    if target:isPzLocked() then end
```
Comment (FakeShinoda) [inline data/events/scripts/creature.lua:83]: if target:isPzLocked() then return true end
this should fix it

Comment (mattyx14) [reply data/events/scripts/creature.lua:83]: no require remove all function only require add condition skull or battle

Comment (lBaah) [reply data/events/scripts/creature.lua:83]: isPzLocked() checks if the player is in fight (pz block), no matter if skulled or not, so it might work

**Review by lBaah — CHANGES_REQUESTED** [review]
Please keep the function and change the line 83: `if target:isPzLocked() then end` to `if target:isPzLocked() then return true end`

**Review by EleanorRigby1 — COMMENTED** [review]
ty

**Review by lBaah — APPROVED** [review]

**Comment by lBaah** [comment]
Thank you for the changes you made.

**Pull request merged**